Chatdown/package.json

```diff
@@ -37,7 +37,7 @@
     "window-size": "^1.1.0"
   },
   "bin": {
-    "chatdown": "./bin/chatdown"
+    "chatdown": "./bin/chatdown.js"
```

Comment:
files in `.bin` should **not** have file extensions. this is due to platform specific and user specific file associations that could conflict. When NPM installs this package, it will create a `chatdown.cmd` which appropriately executes the file on windows. File extensions are omitted with all other packages. open any node_modules/bin and have a look. No one uses file extensions...

Reply:
this is not true.  npm and gulp and many cli's all put .js files into the bin folder.  It's even on the sample page for the npm bin documentation that way.  Are there are objections you have to this delta?